test/pseudo-tty/test-stderr-stdout-handle-sigwinch.js

```diff
@@ -28,3 +28,11 @@ process.stdout._refreshSize = wrap(originalRefreshSizeStdout,
                                    'calling stdout._refreshSize');
 
 process.emit('SIGWINCH');
+
+// In AIX, the child exits even before the python parent
+// can setup the readloop. Provide a reasonable delay.
+if (common.isAix) {
```

Comment:
@jasnell - thanks, I made some changes to this effect, essentially introducing the delay before I/O is performed, instead of after. Please note that from a process control logic perspective, the order of delay is immeterial, while from an I/O logic perspective, the child writes are kept in tact (performed in same uv tick), in order not to affect the intent of the test. I hope you are fine with this?